meta-resin-common/recipes-support/resin-vars/resin-vars/resin-vars

```diff
@@ -87,6 +87,9 @@ if [ -f $CONFIG_PATH ]; then
         # If this was found in the config assume default 443.
         VPN_PORT=443
     fi
+    DNS_SERVERS=${DNS_SERVERS:-8.8.8.8}
```

Comment:
I'm wondering if we should use similar logic for other places as well. such as NTP_SERVERS..